docs/dev/table/streaming/dynamic_tables.zh.md

```diff
@@ -159,31 +159,31 @@ FROM (
 );
 {% endhighlight %}
 
-The [Query Configuration](query_configuration.html) page discusses parameters to control the execution of continuous queries. Some parameters can be used to trade the size of maintained state for result accuracy.
+[查询配置](query_configuration.html)章节讨论了控制连续查询执行的参数。一些参数可以用来在维持状态的大小和获得结果的准确性之间做取舍。
 
-Table to Stream Conversion
+表到流的转换
```

Comment:
I only add a <a> html tag above this to make the anchor link work. 

```html
<a name="table-to-stream-conversion"></a>
```

This is an missing tip in the [Translation Specifications](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications), I just updated the documentation.